metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -534,8 +572,8 @@ def get_lineage_if_enabled(
     def gen_database_key(self, database: str) -> DatabaseKey:
         return DatabaseKey(
             database=database,
-            platform=self.get_underlying_platform(),
-            instance=self.env,
+            platform=self.platform,
+            instance=self.source_config.platform_instance or "",
```

Comment:
@rslanka I couldn't make this work when null, the avro converter complains about a None field in the datum not matching the schema.
I'm not an expert but think this is related to the fact that there is an optional field in an middle class in the inheritance https://github.com/sgomezvillamor/datahub/blob/master/metadata-ingestion/src/datahub/emitter/mcp_builder.py#L36